api/src/main/java/org/openmrs/module/reporting/evaluation/querybuilder/HqlQueryBuilder.java

```diff
@@ -388,20 +393,15 @@ public HqlQueryBuilder where(String propertyName, RangeComparator operator, Obje
 		if (operator != null && value != null) {
 			if (operator.equals(RangeComparator.EQUAL)) {
 				whereEqual(propertyName, value);
-			}
-			else if (operator.equals(RangeComparator.GREATER_EQUAL)) {
```

Comment:
Are these simply formatting changes?

Reply:
Yes, only added [whereNotInAny](https://github.com/cnagadya/openmrs-module-reporting/blob/REPORT-838/api/src/main/java/org/openmrs/module/reporting/evaluation/querybuilder/HqlQueryBuilder.java#L213-L223)

Reply:
Pull requests are harder to review when actual changes are mixed with formatting. If you want to change the formatting, you will need to create a separate pull request. This one should include only the actual changes required by the ticket.

Reply:
All formatting changes have been undone